save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/vulnerability/VulnerabilityStatus.kt

```diff
@@ -20,6 +20,11 @@ enum class VulnerabilityStatus {
      */
     AUTO_APPROVED,
 
+    /**
+     * Created status
+     */
+    CREATED,
+
```

Comment:
Is it clear enough for users and admins that `CREATED` status also requires approve from admins, just like `PENDING_REVIEW`? Is status `CREATED` necessary? Maybe we should rename it somehow and then maybe also `PENDING_REVIEW` status?

Reply:
Statuses are hidden in the system. There are only two states for users: _Approved_ and _Not Approved_.
But it is important for us to distinguish new vulnerabilities from those that have simply been changed.
Changing new ones will not have to create new versions. While old vulnerabilities must have a copy of the version before and after the change.

Reply:
Ok, agreed